app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -34,10 +34,12 @@ export const PressGallery: React.FC = () => {
 
   const [content, setContent] = React.useState<IContentSearchResult[]>([]);
   const [pressMembers, setPressMembers] = React.useState<IPressMember[]>([]);
+  const [initialLoad, setInitialLoad] = React.useState(false);
   const [selected, setSelected] = React.useState<IContentModel[]>([]);
   const [dateOptions, setDateOptions] = React.useState<IDateOptions[]>([]);
   const [aliases, setAliases] = React.useState<string[]>([]);
   const [loading, setLoading] = React.useState(false);
+  const [groupedContent, setGroupedContent] = React.useState<any[]>([]);
```

Comment:
fix

Reply:
Don't use `any`

Reply:
thats what the fix is for :stuck_out_tongue:, closing this PR for now though have an idea

Reply:
(was telling myself to fix)